### DISCUSSION TITLE
Uncomprehensible error message

### DISCUSSION THREAD

**Discussion by jwezel** [optuna/optuna#5377] [Q&A] [answered]
I've got an error message like this on an optimization run:

`Trial 6 failed because of the following error: The number of the values 10 did not match the number of the objectives 1.`

I can't make sense of this and searching for it yielded nothing.

Can somebody explain?

**Comment by nzw0301** [comment]
Could you share the minimal reproducible code with us?

**Comment by jwezel** [comment, marked as answer]
Mistake: I unknowingly returned a list of values from the objective function instead of only one
